django_celery_beat/migrations/0006_periodictask_priority.py

```diff
@@ -7,7 +7,11 @@
 
 class Migration(migrations.Migration):
     dependencies = [
-        ('django_celery_beat', '0005_add_solarschedule_events_choices_squashed_0009_merge_20181012_1416'), # noqa
+        ('django_celery_beat', '0005_add_solarschedule_events_choices'),
+        ('django_celery_beat', '0006_auto_20180210_1226'),
+        ('django_celery_beat', '0006_auto_20180322_0932'),
+        ('django_celery_beat', '0007_auto_20180521_0826'),
+        ('django_celery_beat', '0008_auto_20180914_1922'),
```

Comment:
Please add a comment here about why a 0006 depends on numbers after it, since that's super unconventional and looks like it's wrong at first glance if you didn't know the history here.

Reply:
Comment added. If it's not good enough, please propose some wording. Okay?

Reply:
I'm not sure what BEGIN ex and END ex mean.  

Maybe say something like 

    # depends on higher numbers due to a squashed migration
    # that was later removed due to migration issues it caused

Reply:
Comment added ([as suggested](https://github.com/celery/django-celery-beat/pull/246#issuecomment-489171503))